internal/gcs-sidecar/handlers.go

```diff
@@ -338,7 +338,11 @@ func (b *Bridge) modifySettings(req *request) (err error) {
 
 		case guestresource.ResourceTypeWCOWBlockCims:
 			// This is request to mount the merged cim at given volumeGUID
-			wcowBlockCimMounts := modifyGuestSettingsRequest.Settings.(*guestresource.WCOWBlockCIMMounts)
+			if modifyGuestSettingsRequest.RequestType == guestrequest.RequestTypeRemove {
+				return fmt.Errorf("not implemented")
+			}
```

Comment:
@ambarve Thanks for the changes. What about `RequestTypeAdd`? Is it because the default Request type is Add? I understood from our discussion that you would add a no-op (for now) for both types.

Reply:
The types are either Add or Remove. We do want the current handling to happen for Add. When I add the implementation for Remove (in a separate PR), I will restructure the code a bit better for to have a switch case on the type of request.

Reply:
Ok. I think you want the current handling _only_ on `RequestTypeAdd` type and not on `RequestTypeRemove`. So a switch case is necessary - a separate PR sounds good.